crud/common/utils.lua

```diff
@@ -140,6 +140,15 @@ function utils.unflatten(tuple, space_format)
     return object
 end
 
+function utils.extract_sharding_key(tuple, fieldnos)
```

Comment:
This method is not about sharing key, it just extracts subset of fields from tuple - it is more general purpose function.

Reply:
function works similarly to `extract_key()` so the name is similar too

Reply:
`extract_sharding_key()` has been removed